README.md

```diff
@@ -43,6 +43,59 @@ Now:
 * visit `/hello` in your browser
 
 
+## Reverse routing
+Exprestive exports reverse routing functions to a `paths` object. By default it is saved to `res.locals.paths` for easy access from views.
```

Comment:
While "reverse routing functions" is technically correct, that's a term that doesn't help users of your library understand what they can use them for. "routing helper functions" or "url builder functions" seems easier understandable. Aha, I can use them to create URLs to point to!